graphql/ui_plugin_build_baron.go

```diff
@@ -42,6 +51,17 @@ func BbFileTicket(context context.Context, taskId string) (bool, error) {
 	settings := env.Settings()
 	queue := env.RemoteQueue()
 	buildBaronProjects := BbGetConfig(settings)
+
+	//if there is a custom web-hook, use that
+	annotationSettings := buildBaronProjects[t.Project].TaskAnnotationSettings
```

Comment:
Since you aren't using annotationSettings anywhere, you can drop this variable and just use buildBaronProjects directly at 57

Reply:
True, but I initially did it that way and it was super long. I did that for added clarity and readability.